src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -71,12 +65,14 @@
   JComboBox<PullBeforeOverrideOption> pullBeforeOverrideField;
   JTextPane pullBeforeHintPanel;
   JLabel pullBeforeOverrideLabel;
+  private JCheckBox overwriteExistingFilesCheckBox;
```

Comment:
Make it final. In general, we want everything to be final (immutable) where possible.

Reply:
When I do that I get two compile errors. One on the declaration saying the field may not have been initialized and the second in the $$$setupUI$$$ method saying cannot assign to final field.

Reply:
You're right, sorry!